editor/editor_settings.cpp

```diff
@@ -583,6 +583,9 @@ void EditorSettings::_load_default_text_editor_theme() {
 	_initial_set("text_editor/highlighting/word_highlighted_color", Color(0.8, 0.9, 0.9, 0.15));
 	_initial_set("text_editor/highlighting/search_result_color", Color(0.05, 0.25, 0.05, 1));
 	_initial_set("text_editor/highlighting/search_result_border_color", Color(0.1, 0.45, 0.1, 1));
+
+	// GDScript highlighter
+	_initial_set("text_editor/highlighting/GDScript/function_definition_color", Color::html("#01e1ff"));
```

Comment:
I'm not sure about using uppercase letters in parameter names, we haven't done it so far so it's maybe better to stick to snake_case.